boom/command.py

```
@@ -4709,6 +4709,14 @@ def main(args: List[str]) -> int:
             return 1
         bc = get_boom_config()
 
+    # legacy_enable = False has been the default since BoomConfig was introduced
+    # in boom-0.9: very few installations are expected to remain where this
+    # is enabled.
+    if bc.legacy_enable == True:
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `bc.legacy_enable:` for truth checks

Replace with `bc.legacy_enable`

(E712)